crates/polars-io/src/lib.rs

```diff
@@ -165,7 +165,7 @@ pub(crate) fn finish_reader<R: ArrowReader>(
 }
 
 static CLOUD_URL: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r"^(s3|gs|gs|gcs|file|adbs|az|adl)\:\/\/.+$").unwrap());
+    Lazy::new(|| Regex::new(r"^(s3|gs|gs|gcs|file|abfs|azure|az|adl)\:\/\/.+$").unwrap());
```

Comment:
Maybe also add `local`? 
It's often used by other data processing engines such as Spark or Ray.